src/components/TransactionDialogs/SendDialog/index.tsx

```diff
@@ -0,0 +1,141 @@
+  const {
+    register,
+    handleSubmit,
+    watch,
+    reset,
+    formState: { errors },
+  } = useForm()
+  const iconProps = useIconProps()
+  const { balances } = account
+  const [recipients, setRecipients] = React.useState<
+    Array<{ amount: string; denom: string; address: string }>
+  >([{ amount: '', denom: 'DSM', address: '' }])
+  const [memo, setMemo] = React.useState('')
+
+  return (
+    <Dialog
+      open={open}
+      onClose={() => {
+        onClose(false)
+      }}
+      title={stage}
+    >
+      <div className="flex flex-col">
+        <div className="px-5 flex flex-row items-start">
+          <p>Available amount</p>
+          {balances.available.map((x, y) => {
+            const { amount, denom } = x
+            return (
+              <>
+                <p className="font-bold pl-2">{amount}</p>
+                <p className="font-bold pl-1">{denom}</p>
+              </>
+            )
+          })}
+        </div>
+        <div className="px-5 pt-5 flex flex-row items-center">
+          <div className="w-1/2">
+            <p>Recipient address</p>
+          </div>
+          <div className="w-1/2 pl-2">
+            <p>Amount</p>
+          </div>
+        </div>
+        {recipients.map((v, i) => (
+          <div className="px-5 pb-5 flex flex-row items-center">
+            {recipients.length <= 1 ? null : (
+              <button onClick={() => setRecipients((d) => d.filter((a, j) => j !== i))}>
+                <RemoveIcon className={`${iconProps} mr-2 flex items-center`} key={i} />
+              </button>
+            )}
+            <div className="flex flex-row items-center justify-between w-full">
+              <input
+                className="bg-gray-200 p-1 rounded w-1/2"
+                value={v.address}
+                onChange={(e) =>
+                  setRecipients((rs) =>
+                    rs.map((r, j) => (i === j ? { ...r, address: e.target.value } : r))
+                  )
+                }
+              />
+              <input
+                className="bg-gray-200 p-1 rounded w-1/2 ml-2"
+                value={v.amount}
+                placeholder={v.denom}
+                onChange={(e) =>
+                  setRecipients((rs) =>
+                    rs.map((r, j) => (i === j ? { ...r, amount: e.target.value } : r))
+                  )
+                }
+              />
+            </div>
+          </div>
+        ))}{' '}
+        <button
+          onClick={() => {
+            setRecipients((d) => [...d, { address: '', amount: '', denom: 'DSM' }])
+          }}
+          className="px-5 text-primary-100 hover:opacity-80 text-left"
+        >
+          Add address
+        </button>
+        <div className="ml-5 mt-5">
+          <div className="w-1/2">
+            <p>Memo</p>
+          </div>
+          <input
+            className="bg-gray-200 p-1 rounded text-center w-1/2 h-[10rem]"
```

Comment:
again, why should it be `text-center`?
<img width="268" alt="Screenshot 2022-04-22 at 11 05 21 PM" src="https://user-images.githubusercontent.com/14106911/164741881-26670d55-87ee-4cb3-948f-eabbd8fe105e.png">
?